content/en/docs/concepts/signals/logs.md

```diff
@@ -12,16 +12,17 @@ well-known, widely used logging libraries.
 
 ## OpenTelemetry logs
 
-OpenTelemetry does not define a bespoke API or SDK to create logs. Instead,
-OpenTelemetry logs are the existing logs you already have from a logging
-framework or infrastructure component. OpenTelemetry SDKs and
-autoinstrumentation utilize several components to automatically correlate logs
-with [traces](../traces).
-
-OpenTelemetry's support for logs is designed to be fully compatible with what
-you already have, providing capabilities to wrap those logs with additional
-context and a common toolkit to parse and manipulate logs into a common format
-across many different sources.
+OpenTelemetry provides a Logs API and SDK for producing log records, and
+language SDKs and autoinstrumentation integrate with existing logging
+frameworks. Log records in OpenTelemetry are modeled as events: a timestamped
```

Comment:
I would not call logging bridges "autoinstrumentation" since there is no instrumentation activity, we call them "bridges" or similar.